comictaggerlib/comiccacher.py

```diff
@@ -298,7 +306,7 @@ def get_volume_info(self, volume_id: int) -> CVVolumeResults | None:
             cur.execute("DELETE FROM Volumes WHERE timestamp  < ?", [str(a_week_ago)])
 
             # fetch
-            cur.execute("SELECT id,name,publisher,count_of_issues,start_year FROM Volumes WHERE id = ?", [volume_id])
+            cur.execute("SELECT source_name,id,name,publisher,count_of_issues,start_year FROM Volumes WHERE id = ?", [volume_id]) # Don't need to where soure?
```

Comment:
We do need to add the source to the condition.
right now if we were to add myanimelist through jikan
id: 4003 is https://myanimelist.net/manga/4003/Inviting_Darkness
on comic vine 
id: 4003 is https://comicvine.gamespot.com/green-arrow/4050-4003/